contents/docs/cloud/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: PostHog Cloud
+sidebarTitle: Overview
+sidebar: Docs
+showTitle: true
+---
+
+PostHog Cloud is our hosted and managed version of the PostHog open-source platform. It always runs on the latest release with updates and migrations managed for you.
+
+## Getting started
+
+With PostHog Cloud there is no deployment required. So, the steps we recommend to get started are:
+
+1. [Sign up](https://app.posthog.com/signup?src=cloud-index) for a free PostHog Cloud account.
+2. Begin [integrating PostHog](/docs/integrate) into your applications.
+3. Once integrated you'll see [events](/docs/user-guides/events) coming into PostHog from your app. Also see our [complete guide to event tracking](/docs/tutorials/event-tracking-guide).
+4. With events and [actions](/docs/user-guides/actions) defined you're in a great place to define your first [funnel](/docs/user-guides/funnels).
+
+Once you've defined your first funnel, you're in a great position to start exploring the rest of PostHog.
+
+## Where next?
+
+- [Deploying a Reverse Proxy to Posthog Cloud](/docs/cloud/proxy) provides details on setting up a reverse proxy for situations where you want to use PostHog Cloud but still want tracking data to be sent first to your domain.
+- Our [integration docs](/docs/integrate) provide more details on integrating PostHog into your web, mobile, server, or any other type of application.
+- The [user guides](/docs/user-guides) cover using the PostHog application to gain insights into how end-users use your applications and more.
+- Get involved with the PostHog community on [Slack](/slack) and [GitHub](https://github.com/posthog).
```

Comment:
I'd drop this part - less is more & you cover integration earlier.

Reply:
Removed the integration link.

Reply:
I think the reverse proxy is very optional, so maybe put it at the end?